### ISSUE TITLE
[FEAT]: Add Support for gemini-exp-1206 

### ISSUE THREAD

**Issue by dblinick** [Mintplex-Labs/anything-llm#2788]
### What would you like to see?

Self explanatory - would request support for this model in the Gemini UI pick list.   

Thank you!  Love the platform!

**Comment by DangerousBerries** [comment]
Yes please. I also previously discussed implementing an input override with @timothycarambat . That could solve future delays.

**Comment by ernexto** [comment]
Hey @timothycarambat I see you added "gemini-2.0-flash-exp-1206", but the correct model name is "gemini-2.0-flash-exp"... or am I missing osmething?

**Comment by timothycarambat** [comment]
resolved by https://github.com/Mintplex-Labs/anything-llm/pull/2868

Gemini finally added a `/models` endpoint - so it is no longer static!

**Comment by DangerousBerries** [comment]
> resolved by [#2868](https://github.com/Mintplex-Labs/anything-llm/pull/2868)
> 
> Gemini finally added a `/models` endpoint - so it is no longer static!

gemini-2.0-flash-thinking-exp-01-21 was just released and it's not showing up, do you know why?

**Comment by timothycarambat** [comment]
<img width="343" alt="Image" src="https://github.com/user-attachments/assets/0f2fbd33-4fa7-46ee-ba4b-ce0d70a4b80e" />

Shows for me. This is because you're model listing is cached (we do this to serve responses faster:
https://github.com/Mintplex-Labs/anything-llm/blob/ecc67998f1440200ffc95aafa2105b5ee8bd8f0b/server/utils/AiProviders/gemini/index.js#L70-L81

Delete the `.cached_at` and `models.json` in your `storage/models/gemini` .
On desktop that is here: https://docs.anythingllm.com/installation-desktop/storage

On docker, its wherever you bound the storage to.
